.buildkite/check-graphql-schema.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -ex
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+./yarn-run.sh "update-graphql-schema"
```

Comment:
Doesn't this make the build non-deterministic/non-reproducible if the schema changes in the main repo?

Reply:
Yeah, in which case we should ensure the extension still works because we need to support all instance behaviors for all time forever.

Reply:
It still doesn't seem like a good idea to sacrifice build reproducability for that. It would mean _every_ PR is suddenly failing, even a PR to e.g. revert a commit.
If you just want to be notified of any changes to the schema, maybe set up a Sourcegraph saved search with notifications for the schema file in our main repo?